symbolicationWebService.py

```diff
@@ -78,15 +77,13 @@ def initializeSubprocess(options):
   signal.signal(signal.SIGINT, signal.SIG_IGN)
 
   # Setup logging in the child process
+  if "logPath" in options["Log"]:
+    options["Log"]["logPath"] = os.path.join(options["Log"]["logPath"], "subprocess")
   SetLoggingOptions(options["Log"])
 
   # Create the .SYM cache manager singleton
   gSymFileManager = SymFileManager(options)
 
-  # Prefetch recent symbols
-  gSymFileManager.PrefetchRecentSymbolFiles()
-
-
 def processSymbolicationRequest(rawRequest, remoteIp):
   decodedRequest = json.loads(rawRequest)
```

Comment:
The exception is caught, sent to the caller process, re-raised, handled by `SymbolHandler` and it returns an error to the client.